### PR TITLE
Added missing translations to Turkish language

### PR DESCRIPTION
**Pull request by suleymanozev**

### CODE REVIEW

File: packages/tables/resources/lang/tr/table.php [hunk 1]
```diff
@@ -59,6 +59,31 @@
 
     ],
 
+
+    'bulk_actions' => [
```
Comment (danharrin) [inline packages/tables/resources/lang/tr/table.php:63]: These translations are outdated, they have moved to support/actions

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/tables/resources/lang/tr/table.php (hunk 1 above)
Comment (zepfietje) [reply packages/tables/resources/lang/tr/table.php:63]: They're also still in the `en` translations file. Should they be removed there as well, @danharrin?

Comment (danharrin) [reply packages/tables/resources/lang/tr/table.php:63]: oh, really? that must be a merge problem then, yeah they need removing everywhere

Comment (zepfietje) [reply packages/tables/resources/lang/tr/table.php:63]: https://github.com/filamentphp/filament/pull/2886 @danharrin

**Pull request closed without merging**